### DISCUSSION TITLE
Postgres database support

### DISCUSSION THREAD

**Discussion by fredmonroe** [nautechsystems/nautilus_trader#1654] [Q&A] [answered]
Is the postgres cache meant to be an alternative to redis?
Is there an example of how to set it up, I didn't see one - apologies if I missed it.

**Comment by cjdsellers** [comment, marked as answer]
Yes, its an alternative and probably better as a cache now that we've offloaded I/O write overhead to another thread.

Lack of docs mostly because this is still under development.

**Reply by fredmonroe** [reply]
ok thanks - i’ll watch for the docs - i have to run postgres anyway so this would be a good match for me

if i can help in some way by making an example or something let me know
